### PR TITLE
ci: Enforce no unused imports (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**
Followup to #6060, to address the unused-imports comments in [this review](https://github.com/n8n-io/n8n/pull/6060#pullrequestreview-1397629697).

The combine-imports comments are covered by #6072.

### CODE REVIEW

**Review by netroy — DISMISSED** [review]

File: packages/@n8n_io/eslint-config/base.js [hunk 1]
```diff
@@ -422,6 +427,15 @@ const config = (module.exports = {
...
 		'import/prefer-default-export': 'off',
+
+		// ----------------------------------
+		//         no-unused-imports
+		// ----------------------------------
+
+		/**
+		 * https://github.com/sweepline/eslint-plugin-unused-imports/blob/master/docs/rules/no-unused-imports.md
+		 */
+		'unused-imports/no-unused-imports': 'error',
```
Comment (netroy) [inline packages/@n8n_io/eslint-config/base.js:438]: should we make this a warning during `development` ?

Comment (ivov) [reply packages/@n8n_io/eslint-config/base.js:438]: Good idea!

**Comment by ivov** [comment]
Looks good, as author I cannot approve though.

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@0.226.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.226.1)

**Pull request merged**